jobs/build/refresh-images/Jenkinsfile

```diff
@@ -91,6 +91,35 @@ node('openshift-build-1') {
                         error("Version overrides must start with 'v'")
                     }
                     oit_update_docker_args = "--version ${VERSION_OVERRIDE}"
+
```

Comment:
Please have the check run regardless of value of VERSION_OVERRIDE. i.e. even when user runs with 'auto', they should be prompted to confirm of the version found by `images:query-rpm-version`.

Reply:
So you aren't concerned whether they're inconsistent in the case of auto (since they won't be)

Reply:
Right - the goal is to make sure a human agrees with the version that was detected (and they get the reminder to rebuild the puddle if they have not).